src/ComputeSharp/Properties/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,18 @@
+<linker>
+  <assembly fullname="ComputeSharp">
+    <type fullname="Configuration">
+
+      <!-- COMPUTESHARP_ENABLE_DEBUG_OUTPUT switch -->
+      <method signature="System.Boolean get_IsDebugOutputEnabled()" body="stub" value="false" feature="COMPUTESHARP_ENABLE_DEBUG_OUTPUT" featurevalue="false"/>
+      <method signature="System.Boolean get_IsDebugOutputEnabled()" body="stub" value="true" feature="COMPUTESHARP_ENABLE_DEBUG_OUTPUT" featurevalue="true"/>
```

Comment:
Just so I  understand this better, why? Is this like a general guideline? Is it just that if you're enabling additional options it means you're already given up speed so it's not worth it to trim more, or something else? 🤔

Reply:
The way I see it, you want to support removing the debug-related code which means that you want to hardcode disabling debug output. If debug output is not hardcoded to be off, whether to use it will be decided at runtime.

> it's not worth it to trim more

What would be trimmed if debug output was hardcoded to be enabled? `RuntimeFeature.IsDynamicCode***` [can be hardcoded to either true or false](https://github.com/dotnet/runtime/blob/eaa9717d90115cea43b4cdd7a2a49e6d3c3d780e/src/coreclr/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.xml#L3-L10), but this can actually code. What would be the benefit here?